collector/engine_innodb.go

```diff
@@ -32,6 +32,13 @@ const (
 	engineInnodbStatusQuery = `SHOW ENGINE INNODB STATUS`
 )
 
+var (
+	// 0 queries inside InnoDB, 0 queries in queue
+	// 0 read views open inside InnoDB
+	rQueries = regexp.MustCompile(`(\d+) queries inside InnoDB, (\d+) queries in queue`)
+	rViews   = regexp.MustCompile(`(\d+) read views open inside InnoDB`)
```

Comment:
Let's be consistent with naming.

```suggestion
	queriesRe = regexp.MustCompile(`(\d+) queries inside InnoDB, (\d+) queries in queue`)
	viewsRe   = regexp.MustCompile(`(\d+) read views open inside InnoDB`)
```